src/constraint/encoding.ts

```diff
@@ -239,6 +239,35 @@ export const ENCODING_CONSTRAINTS: EncodingConstraintModel[] = [
       return true;
     }
   },{
+    name: 'scalePropertiesSupportedByChannel',
+    description: 'Not all scale properties are supported by all encoding channels',
+    properties: [Property.CHANNEL, Property.SCALE],
+    allowWildcardForProperties: false, // unsure about this
```

Comment:
(But then manually check if channel is not wildcard.  

(If channel is a wildcard, there is no point checking about this at all)